acceptance-tests/src/test/java/me/atam/atam4jsampleapp/PassingTestAcceptanceTest.java

```diff
@@ -47,7 +58,24 @@ public void givenPassingTest_whenTestsEndpointCalledAfterTestRun_thenOKMessageRe
         );
     }
 
+    /*
+        TODO - Currently this test relies on a junit test that "knows" if it is being run or not.  Once/if we build support for exposing the if
+        a test run is in progress then this test's implementation can change.
+    */
+    @Test
+    public void givenPassingTest_whenTestsEndpointCalledDuringTestRun_thenStatusOfLastRunReturned() throws InterruptedException {
```

Comment:
Please break long lines (lines spanning the default 120 column width in IntelliJ - https://www.jetbrains.com/help/idea/2016.1/code-style-java.html?origin=old_help#d58663e279) for improved code readability.